xrviz/dashboard.py

```diff
@@ -308,7 +317,10 @@ def create_indexed_graph(self, *args):
 
     def create_taps_graph(self, x, y, clear=False):
         """
-        To mark taps
+        This method return a graph to record the taps, when
```

Comment:
"Create an output layer in the graph which responds to taps from the user

Whenever the user taps (or clicks) the graph, a glyph will be overlaid, and a series extracted at that point"